src/shared/components/defaultTooltip/DefaultTooltip.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
-import Tooltip from 'rc-tooltip';
+import Tooltip, {RCTooltip} from 'rc-tooltip';
```

Comment:
one would think that Props would be on Tooltip?

Reply:
had to `rc-tooltip` library to latest version, which has Props in RCTooltip